MobileApplication/MobileApplication/AppShell.xaml

```diff
@@ -2,7 +2,7 @@
 <Shell xmlns="http://xamarin.com/schemas/2014/forms" 
        xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
        xmlns:local="clr-namespace:MobileApplication.Views"
-       xmlns:localTransactions="clr-namespace:MobileApplication.Views.Transactions"
+       xmlns:localTransactions="clr-namespace:MobileApplication.Views.Transaction"
```

Comment:
This definition (line) is unused, isn't it? It can be removed.

Reply:
Yes, going to remove this.